src/components/Input/Input.tsx

```diff
@@ -72,6 +78,9 @@ class Input extends AutoControlledComponent<Extendable<IInputProps>, any> {
     /** Optional Icon to display inside the Input. */
     icon: customPropTypes.itemShorthand,
 
+    /** Shorthand for the input component */
```

Comment:
nit - end with '.'

Reply:
I had the same thought, but I get this weird warning in the tests without the default value:

> Warning: A component is changing a controlled input of type text to be uncontrolled. Input elements should not switch from controlled to uncontrolled (or vice versa). Decide between using a controlled or uncontrolled input element for the lifetime of the component. More info: https://fb.me/react-controlled-components
        in input (created by FelaTheme)
        in FelaTheme (created by Slot)
        in Slot (created by FelaTheme)
        in Ref (created by FelaTheme)
        in div (created by FelaTheme)
        in FelaTheme (created by Slot)

I think it might be a bug in the `AutoControlledComponent` class. Therefore I'd rather keep it even though it definitely looks redundant